quadra/utils/utils.py

```diff
@@ -249,11 +250,17 @@ def finish(
         export_folder: Folder where the deployment models are exported.
     """
     # pylint: disable=unused-argument
-
     if len(logger) > 0 and config.core.get("upload_artifacts"):
         mlflow_logger = get_mlflow_logger(trainer=trainer)
         tensorboard_logger = get_tensorboard_logger(trainer=trainer)
         file_names = ["config.yaml", "config_resolved.yaml", "config_tree.txt", "data/dataset.csv"]
+        if config.trainer.get("precision") == 16:
```

Comment:
Precision may be a string, particularly if we have bfloat this doesn't work, since we have the lightning trainer available can we check the precision from there?